### PR TITLE
PLATUI-1951: Update hmrc-header with latest changes from govuk-header

### PR DESCRIPTION
**Pull request by kyle-bowden**

### CODE REVIEW

File: src/components/header/template.test.js [hunk 1]
```diff
@@ -232,11 +239,4 @@ describe('header', () => {
       expect($('.hmrc-banner > .hmrc-organisation-logo > p.govuk-body-s').text().trim()).toEqual('Cyllid a Thollau EM');
     });
   });
-
-  describe('additional banners block', () => {
```
Comment (oscarduignan) [inline src/components/header/template.test.js]: @kyle-bowden is this removal an unrelated change?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "5.4.0",
+  "version": "5.7.0",
```
Comment (oscarduignan) [inline package.json]: @kyle-bowden needs an `npm install` so package-lock.json gets updated too - but just to check, it's showing a jump of three versions - shouldn't it say 5.5.0? Latest released version is 5.5.0 - though I can see that changelog has 5.6.0 - not sure what's happened here with versions, might want to investigate this before we merge

Comment (oscarduignan) [reply package.json]: I guess this is the "docs changes don't bump versions" stuff maybe, even though we define the version explicitly I guess it's still short circuiting

File: src/components/header/template.test.js (hunk 1 above)
Comment (kyle-bowden) [reply src/components/header/template.test.js]: @oscarduignan thanks for spotting this, this was removed accidentally.

**Review by oscarduignan — APPROVED** [review]
